packages/react/src/components/select/Select.stories.tsx

```diff
@@ -588,5 +497,124 @@ export const VirtualizedSingleselectWithoutGroups = () => {
   const onChange: SelectProps['onChange'] = useCallback(() => {
     // track changes
   }, []);
-  return <Select options={createOptionsForVirtualization()} onChange={onChange} virtualize texts={defaultTexts} />;
+  return (
+    <Select
+      options={getLargeBatchOfUniqueValues(2000)}
+      onChange={onChange}
+      virtualize
+      texts={defaultTexts}
+      id="hds-select-component"
+    />
+  );
+};
+
+export const FocusListenerExample = () => {
+  const [isFocused, setIsFocused] = useState(false);
+
+  const setIsFocusedInHook = useCallback(
+    (focusValue: boolean) => {
+      setIsFocused(focusValue);
+    },
+    [setIsFocused],
+  );
+
+  // need to memoize all or re-rendering would lose selections
+  const memoizedProps = useMemo(() => {
+    const changeTracking: { selectedOptions: Option[] } = {
+      selectedOptions: [],
+    };
+
+    const onChange: SelectProps['onChange'] = (selectedOptions) => {
+      changeTracking.selectedOptions = selectedOptions;
+    };
+
+    const options = getLargeBatchOfUniqueValues(2000);
+
+    return {
+      texts: defaultTexts,
+      onChange,
+      changeTracking,
+      options,
+    };
+  }, []);
+
+  const onFocus: SelectProps['onFocus'] = useCallback(async () => {
+    memoizedProps.texts.error = '';
+    setIsFocusedInHook(true);
+  }, []);
+  const onBlur: SelectProps['onBlur'] = useCallback(async () => {
+    if (!memoizedProps.changeTracking.selectedOptions.length) {
+      memoizedProps.texts.error = 'Select something';
+    }
+    setIsFocusedInHook(false);
+  }, []);
+  const onChange: SelectProps['onChange'] = useCallback(() => {
+    // track changes
+  }, []);
+
+  return (
+    <>
+      <style>
+        {`
+          .focused,
+          .blurred {
+            padding: 10px;
+          }
+
+          .focused {
+            background-color: #defcde;
+          }
+
+          .blurred {
+            background-color: #ececec;
```

Comment:
We should not put any non-brand-colors to examples, as others might consider it as allowed color. Hds color-black-10 would be #e6e6e6